website/blog/2024-08-01-using-rhel-wsl-podman-machine.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Using RHEL as a WSL podman machine
+description: Build image and use it directly in your kubernetes cluster
+authors: [jeffmaury]
+tags: [podman-desktop, podman, rhel, wsl, machine]
+hide_table_of_contents: false
+---
+
+Red Hat provides a tool called image builder that allows developers to build their own custom image of RHEL in a variety of format. Recently, image builder added WSL as a target so you can run RHEL on Windows as a WSL distribution.
+
+This [article](https://developers.redhat.com/articles/2023/11/15/create-customized-rhel-images-wsl-environment) details the steps and actions required to build and run your RHEL WSL image.
+
+The purpose of this article is to describe the options needed for the RHEL WSL distribution so that is can be used as a Podman machine.
```

Comment:
```suggestion
The purpose of this article is to describe the options needed for the RHEL WSL distribution so that it can be used as a Podman machine.
```